src/cmdlib.sh

```diff
@@ -477,20 +480,29 @@ EOF
         srcvirtfs=("-virtfs" "local,id=source,path=${workdir}/src/config,security_model=none,mount_tag=source")
     fi
 
-    rm -f "${workdir}/tmp/rc"
+    local runvm_console
+    runvm_console="${workdir}/tmp/runvm-console.txt"
+    rm -f "${workdir}/tmp/rc" "${runvm_console}" "${cmdoutput}"
 
     #shellcheck disable=SC2086
-    kola qemuexec -m 2048 --auto-cpus -U --workdir none -- -no-reboot -nodefaults -serial stdio \
+    touch "${runvm_console}"
+    tail -qF "${cmdoutput}" &
+    kola qemuexec -m 2048 --auto-cpus -U --workdir none -- -no-reboot -nodefaults -serial file:"${runvm_console}" \
```

Comment:
I sometimes I do want `-serial stdio` here for messing around in the shell (by adding `bash -i` to `init`) in the supermin environment for testing stuff (I actually did that just yesterday when hacking on https://github.com/coreos/coreos-assembler/pull/1398). But anyway, I think it's rare enough that I can just edit the command-line here when I need to do that (or eventually add an env var or something for it).

Reply:
Yep, I've done that too.
Hopefully we'll replace this part with a saner VM using systemd before we need to do that.
But for now I added a little bit of docs to the top which mentions that trick.